### PR TITLE
Add all Saab vehicles to common.js

### LINKED ISSUE #34: Add all Saab vehicles to common.js
Find all Saab vehicles in Appendix A of this document https://www.scca.com/downloads/49336-2020-solo-rules-complete-reduced-2020-03-16/download and add them to https://github.com/Bjorn248/scca_classifier/blob/master/common.js in a similar way to other Makes.

### PR DESCRIPTION
**Pull request by dinukad**
resolve #34

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -2545,6 +2545,218 @@ const allSoloCars = {
...
+      '1981': ['fp'],
+      '1982': ['fp'],
+      '1983': ['fp'],
+      '1984': ['fp'],
+      '1985': ['fp'],
+      '1986': ['fp'],
+      '1987': ['fp'],
+      '1988': ['fp']
+    },
+    '900 Turbo': {
```
Comment (Bjorn248) [inline common.js:2635]: You shouldn't have duplicate keys. You have `900 Turbo` twice. Please merge these two, by adding `fp` to all years or by removing this model and adding `fsp` to the model with `all` for the years.

File: common.js [hunk 2]
```diff
@@ -2545,6 +2545,218 @@ const allSoloCars = {
...
+      '1976': ['ep'],
+      '1977': ['ep'],
+      '1978': ['ep'],
+      '1979': ['ep'],
+      '1980': ['ep']
+    },
+    '96 Sedan (843 cc, 2-stroke)': {
+      'all': ['ep']
+    },
+    '99': {
```
Comment (Bjorn248) [inline common.js:2689]: Resolve duplicate keys, you have `99` twice.

File: common.js [hunk 3]
```diff
@@ -2545,6 +2545,218 @@ const allSoloCars = {
...
+      '1981': ['ep'],
+      '1982': ['ep'],
+      '1983': ['ep'],
+      '1984': ['ep']
+    },
+    '99 EMS': {
+      'all': ['dsp']
+    },
+    '99 Turbo': {
+      'all': ['dsp']
```
Comment (Bjorn248) [inline common.js:2733]: Can add `gs` here since you know all turbo models are eligible for that class.

File: common.js [hunk 4]
```diff
@@ -2545,6 +2545,218 @@ const allSoloCars = {
...
+      '1991': ['dsp'],
+      '1992': ['dsp'],
+      '1993': ['dsp'],
+      '1994': ['dsp'],
+      '1995': ['dsp'],
+      '1996': ['dsp'],
+      '1997': ['dsp'],
+      '1998': ['dsp']
+    },
+    '900 Turbo': {
```
Comment (Bjorn248) [inline common.js:2657]: Can add `gs` here since you know all turbo models are eligble for that class.

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
This one seems to be a bit tricky. If you'd like me to make a separate branch for you to submit your PR to then I can merge the classes there, or if you want to try to fix it yourself you are welcome to. Ideally, we'd want to have as few model definitions as we need. Also, we'll have to look up each model to see if it is a turbo (so then it goes into `gs`) or a non-turbo (so then it goes into `hs`)

**Comment by dinukad** [comment]
@Bjorn248 Added changes as requested. Please review.

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**